code/index.html

```diff
@@ -1,22 +1,179 @@
 <!DOCTYPE html>
-<html>
+<html lang ="en">
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ALVA</title>
+    <link rel="stylesheet" href="./style.css"/>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@200&display=swap" rel="stylesheet">
   </head>
+  
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+    <header class="header">
+      <a href="#">
+        <img class="logo" src="./circle.svg" alt="Alva logo">
+      </a> 
+      <a class="title" href="#">
+        <h1>ALVA</h1>
+      </a>
+      <nav class="navigation"> 
+        <a href="#">About</a>
+        <a href="#">Archive</a>
+        <a href="#">Contact</a>
+      </nav>
+      
+      <button class="ham">
+        <span class="hamburger"> 
+          <span></span>
+          <span></span>
+          <span></span>
+        </span>
+      </button>
+      <div class="social-media">      <!-- maybe put a class in each of these to regulate the space between them? -->
+        <a href="#" target="_blank">
+          <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24" fill="none" stroke="currentColor" 
+          stroke-width="1" stroke-linecap="round" stroke-linejoin="round" class="feather feather-facebook">
+            <path d="M18 2h-3a5 5 0 0 0-5 5v3H7v4h3v8h4v-8h3l1-4h-4V7a1 1 0 0 1 1-1h3z"></path>
+          </svg>
+        </a> 
+        <a href="#" target="_blank">
+          <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24" fill="none" stroke="currentColor" 
+          stroke-width="1" stroke-linecap="round" stroke-linejoin="round" class="feather feather-instagram">
+            <rect x="2" y="2" width="20" height="20" rx="5" ry="5"></rect>
+            <path d="M16 11.37A4 4 0 1 1 12.63 8 4 4 0 0 1 16 11.37z"></path>
+            <line x1="17.5" y1="6.5" x2="17.51" y2="6.5"></line>
+          </svg>
+        </a>
+        <a href="#" target="_blank">
+          <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24" fill="none" stroke="currentColor" 
+          stroke-width="1" stroke-linecap="round" stroke-linejoin="round" class="feather feather-youtube">
+            <path d="M22.54 6.42a2.78 2.78 0 0 0-1.94-2C18.88 4 12 4 12 4s-6.88 0-8.6.46a2.78 2.78 0 0 0-1.94 2A29 29 0 0 0 1 
+            11.75a29 29 0 0 0 .46 5.33A2.78 2.78 0 0 0 3.4 19c1.72.46 8.6.46 8.6.46s6.88 0 8.6-.46a2.78 2.78 0 0 0 1.94-2 29 
+            29 0 0 0 .46-5.25 29 29 0 0 0-.46-5.33z"></path>
+            <polygon points="9.75 15.02 15.5 11.75 9.75 8.48 9.75 15.02"></polygon>
+          </svg>
```

Comment:
Nice with `svg`s, make it look super crisp!

Reply:
You could display them as `img` as you did with the circle, but I think it's fine like this as well